### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by westonplatter**

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @westonplatter,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

**Comment by benmccann** [comment]
there is a thing called Anorm: https://www.playframework.com/documentation/2.4.x/Anorm

you could potentially capitalize it if you wanted to reduce confusion

**Comment by westonplatter** [comment]
@benmccann thanks for the input and link to Anorm. I reverted the text, capitalized, and added a needed comma.

File: documentation/manual/detailedTopics/database/Developing-with-the-H2-Database.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!--- Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com> -->
 # H2 database
 
-The H2 in memory database is very convenient for development because your evolutions are run from scratch when play is restarted.  If you are using anorm you probably need it to closely mimic your planned production database.  To tell h2 that you want to mimic a particular database you add a parameter to the database url in your application.conf file, for example:
+The H2 in memory database is very convenient for development because your evolutions are run from scratch when play is restarted.  If you are using Anorm, you probably need it to closely mimic your planned production database.  To tell h2 that you want to mimic a particular database you add a parameter to the database url in your application.conf file, for example:
```
Comment (benmccann) [inline documentation/manual/detailedTopics/database/Developing-with-the-H2-Database.md]: >To tell h2 that you want to mimic a particular database you add a parameter to the database url in your application.conf file, for example:

Could probably capitalize H2, put a comma after database, and change the existing comma to a period

**Comment by benmccann** [comment]
I'm reading this a bit more closely now and debating whether it was meant to be "an orm" like you originally suggested or "Anorm". Really it doesn't make much sense to me either way. Probably "an orm" makes more sense than "Anorm", but I wouldn't be against removing that entire sentence since it adds very little and doesn't explain why you'd want it in that particular case and less in others.

**Comment by jroper** [comment]
I think it's meant to be Anorm.  In Anorm, you write all your SQL statements yourself - there's no indirection between your code and the databases dialect, so your SQL statements are likely to be specific to your production database, which is why you need H2 to closely mimic the dialect of your production database.  This is in contrast to an ORM, which usually abstracts the databases dialect away from you so you can run the same code against multiple databases.  In this case, H2 can just use its own native dialect, and you configure the ORM to speak H2.

**Pull request merged**